applications/ShapeOptimizationApplication/tests/algorithm_penalized_projection_test/parameters.json

```diff
@@ -63,19 +63,10 @@
         },
         "output" : {
             "design_output_mode" : "WriteDesignSurface",
```

Comment:
Shouldn't now in all unit test "design_output_mode" be "None"? Or am I missing something? I mean that was the original reason to introduce this option. If yes, then it is missing in all algorithm tests.

Reply:
I left one gid and one vtk output active on purpose, such that they are tested as well

Reply:
You could add a test if it is written correctly (maybe also add it to the tests of the `DesingLogger`)
otherwise all this does it check that there are no exceptions

there are [functionalities in the core](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_scripts/compare_two_files_check_process.py#L88-L104) to compare output files